docs/automated-tests/test-execution/quick-run-browsers.md

```diff
@@ -1,26 +1,36 @@
 ---
 title: Browser Control in Quick Execution
 page_title: Browser Control in Quick Execution
-description: "Test Studio Quick test Execution. How to Set preferred browser for test runs and test recording. Wuick Access the browser calibration tool"
+description: "Learn how to set a preferred browser for Test Studio's quick test execution and how to calibrate a browser."
 position: 2
 ---
 # Browser Control in Quick Execution
 
-The **Test Ribbon** in Test Studio allows you to trigger a quick test execution. Along with that it provides quick access to some useful project settings.
```

Comment:
- Use bold only for the name of the UI element - **Test**
- You can omit _Along with that_, this expression doesn't add value and only makes the sentence longer.